lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -757,6 +764,8 @@ def _rows_for_users(self, context, users, header_row, graded_scorable_blocks):
         """
         Returns a list of rows for the given users for this report.
         """
+        if six.text_type(context.course_id) == 'course-v1:MITx+CTL.SC0x+1T2019':
+            context.update_status('ProblemGradeReport Investigation log: Start of row creation for users')
         success_rows, error_rows = [], []
         for student, course_grade, error in CourseGradeFactory().iter(users, context.course):
             student_fields = [getattr(student, field_name) for field_name in header_row]
```

Comment:
Let's have one log inside this loop too, b/c this will execute after all first iteration of iterators.